packages/web/Cargo.toml

```diff
@@ -12,6 +12,8 @@ keywords = ["dom", "ui", "gui", "react", "wasm"]
 [dependencies]
 dioxus-core = { workspace = true }
 dioxus-html = { workspace = true, features = ["wasm-bind"] }
+dioxus-hot-reload = { workspace = true, features = ["client"] }
```

Comment:
Trying to run web platform gave me this error.
Need to enable feature for CloseEvent for HotReload.

Cargo.toml for web package.
Under features section for hot_reload add `"web-sys/CloseEvent"`

```
error[E0432]: unresolved import `web_sys::CloseEvent`                             
  --> packages\web\src\hot_reload.rs:15:23                                        
   |                                                                              
15 | use web_sys::{window, CloseEvent, MessageEvent, WebSocket};                  
   |                       ^^^^^^^^^^                                             
   |                       |                                                      
   |                       no `CloseEvent` in the root                            
   |                       help: a similar name exists in the module: `MouseEvent`
```